tests/hash-graph-test-data/rust/src/data_type/value.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type",
+  "kind": "dataType",
+  "$id": "https://blockprotocol.org/@blockprotocol/types/data-type/value/v/1",
+  "title": "Value",
+  "description": "A value that can be stored in a graph",
+  "anyOf": [{ "type": "null" }]
```

Comment:
Why not the full list here? I'd call this 'Null' to avoid confusion if it's just for testing